src/System.Private.CoreLib/shared/System/Threading/Tasks/TaskContinuation.cs

```diff
@@ -818,30 +816,6 @@ internal static void UnsafeScheduleAction(Action action, Task task)
             ThreadPool.UnsafeQueueUserWorkItemInternal(atc, preferLocal: true);
         }
 
-        /// <summary>Throws the exception asynchronously on the ThreadPool.</summary>
```

Comment:
Good that we have just one copy of the method now. The dependencies generally go from the AsyncMethodBuilders to Task, not the other way around. It would look better to keep this method on the Task side of things. Maybe move the method to Task.